app/controllers/email_alert_subscriptions_controller.rb

```diff
@@ -25,20 +25,24 @@ def render_error(error_message)
     render action: :new
   end
 
-  def content
-    @content ||= ContentItem.from_content_store(request.path).as_hash
+  def content_item
+    @content_item ||= ContentItem.from_content_store(request.path).as_hash
   end
 
   def signup_presenter
-    @signup_presenter ||= SignupPresenter.new(content, params)
+    @signup_presenter ||= SignupPresenter.new(content_item, params)
+  end
+
+  def selected_taxon
```

Comment:
We should move this into the `SignupPresenter`, since it's a presentation concern.